test/socialAuth.spec.js

```diff
@@ -0,0 +1,67 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../server';
+import { user } from './mockStrategy';
+import { userEmailValue } from './fixtures';
+
+chai.use(chaiHttp);
+
+describe('GET Social Authentication', () => {
+  describe('GET Social Auth Callback', () => {
+    it('should save a twitter user to database', done => {
+      chai
+        .request(server)
+        .get('/api/auth/twitter/callback')
+        .end((err, res) => {
+          expect(res).to.have.status(200);
+          expect(res.body.username).to.equal(user.displayName);
+          expect(res.body.email).to.equal(user.emails[0].value);
+          expect(res.body.image).to.equal(user.photos[0].value);
```

Comment:
Once the mock strategy starts using the provider names, you could also test that the returned socialProvider is correct